File: Informatyka sem 2 2016/Labolatorium 11 Zadanie 2/Zadanie2/src/zadanie3/Shape.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package zadanie3;

import static java.lang.System.out;

/**
 *
 * @author Lenovo
 */
abstract public class Shape 
{
    private String name;
    public Shape(String n)
    {
        name=n;
    }
    public void setName(String n)
    {
        name=n;
    }
    public String getName()
    {
        return name;
    }
    @Override
    public String toString()
    {
        return "Pole obiektu "+getName()+" wynosi: "+area();
    }
    abstract public double area();
}

File: Informatyka sem 2 2016/Labolatorium 3 Zadanie 1/Labolatorium 3 Zadanie 1/Shape.cpp
#include "Shape.h"
#include<iostream>
using namespace std;
Shape::Shape()
{
	cout << "Konstruktor klasy Shape" << endl;
}
Shape::Shape(string n) :name(n)
{
	cout << "Konstruktor klasy Shape" << endl;
}


Shape::~Shape()
{
	cout << "Destruktor klasy Shape" << endl;
}
double Shape::Area()
{
	return 0;
}